File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.java
class Solution {
    public String solution(String str1, String str2) {
        String answer = "";
        int cnt = 1;
        
        System.out.println(1/2);
        
        for(int i = 0; i < str1.length() * 2; i++) {
            if(cnt % 2 != 0) answer += str1.charAt(cnt++/2);
            else answer += str2.charAt(cnt++/2-1);
        }
        
        
        return answer;
    }
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.cs
using System;
using System.Text;

public class Solution {
    public string solution(string str1, string str2) {
        
        StringBuilder sb = new StringBuilder();
        char[] arr1 = str1.ToCharArray();
        char[] arr2 = str2.ToCharArray();
        
        for(int i = 0; i < arr1.Length; i++)
        {
            sb.Append($"{arr1[i]}{arr2[i]}");
        }
        
        return sb.ToString();
    }
}